Content.Server/GameTicking/Rules/PiratesRuleSystem.cs

```diff
@@ -235,6 +244,24 @@ public void MakePirate(Mind.Mind mind)
         if (!mind.OwnedEntity.HasValue)
             return;
         SetOutfitCommand.SetOutfit(mind.OwnedEntity.Value, "PirateGear", EntityManager);
+
+        var pirateRule = EntityQuery<PiratesRuleComponent>().FirstOrDefault();
+        if (pirateRule == null)
+        {
+            //todo fuck me this shit is awful
+            GameTicker.StartGameRule("Pirates", out var ruleEntity);
+            pirateRule = Comp<PiratesRuleComponent>(ruleEntity);
+        }
+
+        // Notificate every player about a pirate antagonist role with sound
+        if (mind.Session != null)
+        {
+            _audio.PlayGlobal(
+                pirateRule.PirateAlertSound,
+                Filter.SinglePlayer(mind.Session),
```

Comment:
can't you just give it the session without the need of a filter (since IPlayerSession inherits ICommonSession)?